DQMOffline/Trigger/python/B2GMonitoring_cff.py

```diff
@@ -1,4 +1,88 @@
 import FWCore.ParameterSet.Config as cms
 
+# THIS WILL BE UPDATED WHEN 19178 IS MERGED:
+#from DQMOffline.Trigger.JetMETMonitor_cfi import hltJetMETmonitoring
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+from DQMOffline.Trigger.B2GTnPMonitor_cfi import B2GegmGsfElectronIDsForDQM,B2GegHLTDQMOfflineTnPSource
+
+# B2G triggers:
+#HLT_AK8PFHT750_TrimMass50_v*
+#HLT_AK8PFJet380_TrimMass30_v*
+#HLT_AK8PFHT800_TrimMass50_v*
+#HLT_AK8PFJet400_TrimMass30_v*
+#HLT_AK8PFHT850_TrimMass50_v*
+#HLT_AK8PFJet420_TrimMass30_v*
+#HLT_AK8PFHT900_TrimMass50_v*
+# HLT_AK8PFHT700_TrimR0p1PT0p03Mass50                                                                                                                                         
+AK8PFHT750_TrimMass50_HTmonitoring = hltHTmonitoring.clone()
+AK8PFHT750_TrimMass50_HTmonitoring.FolderName = cms.string('HLT/B2GMonitor/AK8PFHT750_TrimMass50')
+AK8PFHT750_TrimMass50_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_AK8PFHT750_TrimMass50_v*")
```

Comment:
These are analysis dependent. I would say that the HLT monitoring we want should be more directly related to the trigger than to individual analyses, no?